examples/AlwaysSucceeds.purs

```diff
@@ -29,18 +28,18 @@ import Contract.Transaction
 import Contract.TxConstraints as Constraints
 import Contract.Utxos (utxosAt)
 import Contract.Value as Value
-import Contract.Wallet (mkNamiWalletAff)
 import Data.BigInt as BigInt
 import Data.Map as Map
-import Effect.Aff (delay)
+import Effect.Aff (delay, launchAff_)
 import Plutus.Types.Transaction (UtxoM(UtxoM))
-import Types.Scripts (ValidatorHash)
+import Types.Scripts (Validator, ValidatorHash)
 import Types.Transaction (TransactionInput(TransactionInput), TransactionHash)
 import Types.TxConstraints (TxConstraints)
+import Wallet (mkGeroWalletAff)
 
 main :: Effect Unit
 main = launchAff_ $ do
-  wallet <- Just <$> mkNamiWalletAff
+  wallet <- Just <$> mkGeroWalletAff
```

Comment:
I'd like to keep all examples using Nami, since users are more likely to have that installed